src/test/java/step2/domain/RankTest.java

```diff
@@ -8,15 +8,38 @@
 public class RankTest {
 
     @Test
-    @DisplayName("매칭된 번호 갯수에 따른 등수 체크")
-    public void rankCheck() {
+    @DisplayName("1등 체크")
+    public void firstRankCheck() {
         assertEquals(Rank.getRank(6, false), Rank.FIRST);
+    }
+
+    @Test
+    @DisplayName("2등 체크")
+    public void SecondRankCheck() {
```

Comment:
테스트 메서드의 시작이 대문자로 하고 있네요 :)
뿐더러 메서드는 왠만하면 동사구로 시작해주시면 좋을 것 같아요!
```suggestion
    public void checkSecondRank() {
```